src/ameba/reportable.cr

```diff
@@ -5,35 +5,37 @@ module Ameba
     getter issues = [] of Issue
 
     # Adds a new issue to the list of issues.
-    def add_issue(rule, location : Crystal::Location?, end_location : Crystal::Location?, message, status = nil)
+    def add_issue(rule, location, end_location, message, status = nil) : Issue
```

Comment:
do we rollback https://github.com/crystal-ameba/ameba/pull/184 ?

Reply:
Yep, it's not needed anymoar since after this change there's no splatted arguments involved.